src/schemas/28-1900/schema.js

```diff
@@ -0,0 +1,114 @@
+import constants from '../../common/constants';
+import _ from 'lodash';
+import definitions from '../../common/definitions';
+import schemaHelpers from '../../common/schema-helpers';
+
+let schema = {
+  $schema: 'http://json-schema.org/draft-04/schema#',
+  title: 'DISABLED VETERANS APPLICATION FOR VOCATIONAL REHABILITATION (28-1900)',
+	type: 'object',
+  additionalProperties: false,
+  definitions: _.pick(definitions, [
+    'fullName',
+    'address',
+    'phone',
+    'ssn',
+    'date',
+    'dateRange'
+  ]),
+  properties: {
+    veteranFullName: {
+      $ref: '#/definitions/fullName'
+    },
+    veteranSocialSecurityNumber: {
+      $ref: '#/definitions/ssn'
+    },
+    veteranDateOfBirth: {
+      $ref: '#/definitions/date'
+    },
+    veteranAddress: {
+      $ref: '#/definitions/address'
+    },
+    newVeteranAddress: {
+      $ref: '#/definitions/address'
+    },
+    homePhone: {
+      $ref: '#/definitions/phone'
+    },
+    mobilePhone: {
+      $ref: '#/definitions/phone'
+    },
+    email: {
+      type: 'string',
+      format: 'email'
+    },
+    vaRecordsOffice: {
+      type: 'string',
+    },
+    yearsOfEducation: {
+      type: 'number',
+    },
+    previousPrograms: {
+      type: 'array',
+      items: {
+        type: 'object',
+        additionalProperties: false,
```

Comment:
We don't usually use this outside the top level in our schemas (though I think it's set globally whenever we validate, maybe?).